### DISCUSSION TITLE
Need to set "OR" combine instead of "AND" combine in native Filter

### DISCUSSION THREAD

**Discussion by NattikaNJ** [apache/superset#26134] [Q&A / Help]
In the past, Superset allowed me to use a Filter Chart on dashboards. With it, I could easily set up multiple filters using the "OR" combination through a Jinja Template. This filter chart was a great tool for my dashboards.

However, the Filter Chart has been deprecated, leaving only the native filter, which works with an 'AND' combination.

![Capture](https://github.com/apache/superset/assets/32167235/1a2b945d-3f1f-4aad-80b4-7d995c56c5f2)

I need to modify the where-clause to switch from 'AND' to 'OR'.
Does anyone know how to solve this issue?

Thank you in advance!

**Comment by dkrat7** [comment]
I think this is solvable with jinja (filter has to be removed from outer query). There is some additional work, because you have to put filters in inner query (virtual dataset), something like this:

```
SELECT * FROM mytable
WHERE col1 IN {{ filter_values('filter1', remove_filter=True))|where_in }}
OR col2 IN {{ filter_values('filter2', remove_filter=True))|where_in }}
```

**Reply by NattikaNJ** [reply]
Thank you, yes it works!

**Reply by PratikshitSingh** [reply]
Hi @dkrat7  @NattikaNJ - I'm trying to get "OR" conditions on my filters something similar to the above usecase, but only using APIs. Can you please help explain if you worked with API payload to make such changes that I can use?

**Comment by gauravganna** [comment]
It works!
